.storybook/storyshots.test.js

```diff
@@ -0,0 +1,18 @@
+import initStoryshots from '@storybook/addon-storyshots';
+
+describe('', () => {
+  // Our localhosts have a different locale than CI. Since the snapshots
+  // contain localized dates (eg. TxReceipt), we stub the call and neutralize
+  // localization.
+  beforeAll(() => {
+    jest.spyOn(global.Date.prototype, 'toLocaleString').mockImplementation(function (args) {
+      return new Date(this.valueOf()).toISOString();
+    });
+  });
+
+  initStoryshots();
+
```

Comment:
Is there a way to verify that all our stories are tested? I seem to remember from my work on this that some weren't being run at one point.

Reply:
When I create an empty `.stories` file, I get a `untested stories` warning in the console.
Looking at the docs I didn't find another option. I doubt we can do better.